src/ProfilesList.json

```
@@ -329,5 +329,6 @@
   "roy2pushkar.json",
   "blmayer.json",
   "nandha46.json",
-  "VishalIsharani.json"
-]
\ No newline at end of file
+  "VishalIsharani.json",
+  reallywasi.json,
```

```suggestion
  "reallywasi.json"
```
Put it inside the `" "` and if it in last don't add comma.